pennylane/devices/null_qubit.py

```diff
@@ -190,7 +190,9 @@ class NullQubit(Device):
             (``['aux_wire', 'q1', 'q2']``). Default ``None`` if not specified.
         shots (int, Sequence[int], Sequence[Union[int, Sequence[int]]]): The default number of shots
             to use in executions involving this device.
-        track_resources (bool): If True, track the number of resources used by the device and save them to a JSON file. This argument is experimental and subject to change.
+        track_resources (bool): If True, track the number of resources used by the device and save them to a JSON file.
+            The filename will match "__pennylane_resources_data_*.json" where the glob is a timestamp in nanoseconds 
```

Comment:
Is "glob" a standard enough concept to use it here? I don't know.

What is the timestamp referring to? Time of writing the file, starting/ending execution? :) 
I suppose it's anyways not really human-readable (right?) but still might be nice to be precise about it.

Reply:
Updated